### DISCUSSION TITLE
Hotspot not working - not broadcasting the SSID but seeing other changes made

### DISCUSSION THREAD

**Discussion by AlexWilsonsBlog** [WLAN-Pi/feedback#61] [Hardware - WLAN Pi Neo2]
NEO2, fresh image of wlanpi-neo2-v2.1.0-230410.img.

Followed config options for the hotspot. Set the SSID to our preferred choice, made sure all options were turned on as referenced on the https://wlanpi.gitbook.io/cookbook/hotspot-mode site. Some difference in folder names but muddled through that. 

Changed the DHCP settings to use a different range. All is correct. 

Booted to hotspot. I see the channel change to 165. I do not see the SSID. On the unit when checking the WLAN interface, it shows "False" for the SSID. Slight issue...

What am I missing? It used to work.

**Comment by joshschmelzle** [comment]
We no longer support the NEO2 platform.
